### DISCUSSION TITLE
voting UI design

### DISCUSSION THREAD

**Discussion by Flottegurke** [The-Bug-Bashers/VaGABfS#31] [Ideas]
Most of the basic functions are finished by now, so we should be able to start planning the voting functions.

The first question is how are we gonna design the UI for the votings? It should be as easy as possible to vote and see the results.

If you got any ideas, please post them as a comment.

**Comment by dagadeta** [comment]
I think, it should work something like this:

**In the group**
- As an admin or moderator, use the command `voting` to make a voting. Add `"<Voting description>"` behind the command to add a description to the voting.

**In the private chat with the bot**
- Use the command `currentVotings` to see which votings are currently running. The bot will give you a list with the IDs and the descriptions of the votings.
- Use the command `vote <id> <yes/no/abstention>` to vote. (Maybe you could also say which answer options are available for the voting when making it)
- Use the command `voteInfo <id>` to see the current status of the results of the vote you're choosing in a graph.

I think, that it is much more pleasant for the group members if you let the bot do as few as possible important things inside the group and let it communicate directly with them. Maybe it would also be good to let the bot remind you to vote like one day before the voting ends.

After the voting has ended, it sends a message into the group chat and shows the results in a graph.

**Reply by Flottegurke** [reply]
That seems like a good idea

**Comment by Dinkelbroetchen** [comment]
Do you mean a „good“ Idea? <(˶ᵔᵕᵔ˶)>

**Reply by Flottegurke** [reply]
yes
